packages/lodestar-cli/src/cmds/beacon/options.ts

```diff
@@ -14,9 +13,10 @@ export type IBeaconOptions =
     templateConfigFile?: string;
     genesisStateFile?: string;
     testnet?: TestnetName;
+    logFile?: string;
   };
 
-export const beaconOptions = {
+export const beaconOptions: {[k: string]: Options} = {
```

Comment:
By doing this you lose access to each option in a type safe way in other parts of the code such as `beaconOptions.genesisStateFile`

Reply:
how so?
We are passing this variable to yargs.options so all commands are using IBeaconOptions interface anyways.
This change just adds more typesafety by forcing every option to have type Options. Without this you can submit (as I did with typo) invalid option that fails at runtime